compliance/compliance.go

```diff
@@ -100,6 +100,9 @@ type Test struct {
 	// RequiresImplicitReplace marks a test that requires the implementation of AFTOperation
 	// ADD for entries that already exist.
 	RequiresImplicitReplace bool
+	// RequiresIdempotentDelete marks a test that requires the implementation of AFTOperation
+	// DELETE for entries that do not exist.
+	RequiresIdempotentDelete bool
```

Comment:
These flags are being used by openconfig/featureprofiles gribi compliance tests to group sections of tests together - i.e. right now it is unlikely any implementation will pass these test cases because this is a recent protocol spec change.  This flag lets us group/isolate these tests as "known issues" tied to bugs.  Should we try to do this another way?  I was planning on using this to toggle a deviation flag for devices that support most tests but not this one feature.

Reply:
ack, we can talk offline. thanks! LGTM.